add skjold to precommit hooks

omarudolley opened the pull request.

omarudolley committed:
add skjold to precommit hooks

joakimnordling commented on backend/pyproject.toml:
```diff
@@ -8,14 +8,13 @@ packages = [{include = "app", from="."}]
+httpx = "^0.23.0"
 mock = "^4.0.2"
 python-multipart = "^0.0.5"
-pyjwt = "^2.1.0"
```
Since we in the code actually use (py)jwt directly (not only from pyjwt-key-fetcher), I'd prefer to keep it here to indicate we require it.

joakimnordling commented on pyproject.toml:
```diff
@@ -0,0 +1,17 @@
+description = ""
+
+[tool.poetry.dependencies]
+python = "^3.7"
```
Could we update this to require python 3.7 so it's in line with the version required in the backend directory?

joakimnordling commented on pyproject.toml:
```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "mycompany-consent-demo"
+version = "0.1.0"
+description = ""
```
Could we add a description, authors and license here (can be copied mostly from the file in the backend folder, but leave out the `backend` part of the description)?

joakimnordling commented on pyproject.toml:
```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "mycompany-consent-demo"
+version = "0.1.0"
```
Maybe update this to 1.0.0 to match the version used in the backend folder.

omarudolley committed:
add pyjwt

Commit:
Fix issues found in review

Commit:
Update firedantic

Fixes this issue:
```
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).
```

joakimnordling reviewed (approved):
LGTM
